### PR TITLE
update seek bar

### PR DESCRIPTION
**Pull request by m-derakhshan**
Disable the system gesture near the seek bar.
You need to exclude system gestures near the seek bar area. Otherwise, the os will give a higher priority to
system gestures and the user won't be able to perform drag.
You can see the effect in the before and after video.

https://github.com/user-attachments/assets/04a562fe-3509-4f53-bf09-147a41193c3e

https://github.com/user-attachments/assets/7b0bf4e1-fe68-4130-b620-a9149fe52137

### CODE REVIEW

**Comment by m-derakhshan** [comment]
Hello,
I would appreciate it if you could check this.
Thank.
@ceyhun-o @georgechoustoulakis  @MattiasBuelens  @tvanlaerhoven

**Comment by ceyhun-o** [comment]
Hi @m-derakhshan,

Thank you for your contribution!

We will review the PR as soon as possible.

**Comment by MattiasBuelens** [comment]
Apologies, I clicked the wrong button. 😅 We're still reviewing!

File: ui/src/main/java/com/theoplayer/android/ui/SeekBar.kt [hunk 1]
```diff
@@ -42,7 +45,12 @@ fun SeekBar(
     var wasPlayingBeforeSeek by remember { mutableStateOf(false) }
 
     Slider(
-        modifier = modifier,
+        modifier = modifier.systemGestureExclusion {
+            Rect(
+                topLeft = Offset(0f, 0f),
+                bottomRight = Offset(it.size.width.toFloat(), it.size.height.toFloat() + 20)
```
Comment (MattiasBuelens) [inline ui/src/main/java/com/theoplayer/android/ui/SeekBar.kt:51]: Can you elaborate on the `+ 20`? Where does this number come from?

Comment (m-derakhshan) [reply ui/src/main/java/com/theoplayer/android/ui/SeekBar.kt:51]: Hello.
Thanks for pointing it out. Yes, you're right. In our case, we added this as a safe margin equal to the size of the material slider thumbnail. You may remove it. @MattiasBuelens

**Pull request merged**